packages/api/src/functions/dbAuth/DbAuthHandler.ts

```diff
@@ -109,7 +109,7 @@ interface DbAuthHandlerOptions {
      * were included in the object given to the `signUp()` function you got
      * from `useAuth()`
      */
-    handler: (signupHandlerOptions: SignupHandlerOptions) => Promise<any>
+    handler: (signupHandlerOptions: SignupHandlerArgs) => any
```

Comment:
Again, this is a genuine any.

Reply:
>  Whatever you want to happen to your data on new user signup. Redwood will
 check for duplicate usernames before calling this handler. At a minimum
 you need to save the `username`, `hashedPassword` and `salt` to your
 user table. `userAttributes` contains any additional object members that
 were included in the object given to the `signUp()` function you got
 from `useAuth()`.

 If you want the user to be immediately logged in, return the user that
 was created.

 If this handler throws an error, it will be returned by the `signUp()`
 function in the form of: `{ error: 'Error message' }`.

 If this returns anything else, it will be returned by the
 `signUp()` function in the form of: `{ message: 'String here' }`